tests/assets/crs/mlflowserver.yaml

```diff
@@ -7,11 +7,13 @@ spec:
   name: wines
   resources:
     requests:
-      memory: "64Mi"
+      memory: "500Mi"
       cpu: "500m"
+      ephemeral-storage: "2G"
     limits:
-      memory: "128Mi"
+      memory: "1G"
```

Comment:
do we still need 1G ?
or it could be lower?

if resource is freed after test, then we probably can ignore it

Reply:
Yes. Resouces are freed after every test.